src/ManageLiquidity/RemoveLiquidity.js

```diff
@@ -101,15 +101,15 @@ export default function RemoveLiquidity(props) {
       <Form.Group className="basic">
         <Form.Label>{token.name} to Remove</Form.Label>
         <div>
-          {value((percentage.value * tokensInPool) / (BASE_FACTOR * 100n))}
-          of {value(tokensInPool)}
+          {value((percentage.value * tokensInPool) / (BASE_FACTOR * 100n))} of{" "}
+          {value(tokensInPool)}
         </div>
         <hr className="mt-0" />
       </Form.Group>
       <Form.Group className="basic">
         <Form.Label>USD To Remove</Form.Label>
         <div>
-          {value((percentage.value * baseTokensInPool) / (BASE_FACTOR * 100n))}
+          {value((percentage.value * baseTokensInPool) / (BASE_FACTOR * 100n))}{" "}
```

Comment:
prettier wants them to be like this for some reason, symmetrical though the two changes are not